### PR TITLE
Modify byron-to-mary/mkfiles.sh to allow starting nodes in specific era

### PR DESCRIPTION
**Pull request by Jimbo4350**
You can also start a cluster in the Shelley, Allegra or Mary era by supplying an argument to `mk-files.sh`.

```bash
./scripts/byron-to-mary/mk-files.sh shelley # Start nodes in Shelley era
./scripts/byron-to-mary/mk-files.sh allegra # Start nodes in Allegra era
./scripts/byron-to-mary/mk-files.sh mary    # Start nodes in Mary era
```

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -12,11 +12,16 @@ Contains all the scripts relevant to benchmarking `cardano-node`. See the benchm
...
-Contains a script that sets up a cluster beginning in the Byron era and can transition to the Shelley era.
-
+#### byron-shelley-allegra-mary
+Contains a script that sets up a cluster beginning in the Byron era and can transition to the Shelley era. You can also start a cluster in the Shelley, Allegra or Mary era by supplying an argument to `mk-files.sh`.
+E.g
+```bash
+./mk-files.sh shelley # Starts nodes in Shelley era
+./mk-files.sh allegra # Starts nodes in Allegra era
+./mk-files.sh mary    # Starts nodes in Mary era
+```
```
Comment (newhoggy) [inline scripts/README.md:22]: Seems like it should be run with:

```
./scripts/byron-to-mary/mkfiles.sh mary
```

instead.

**Comment by newhoggy** [comment]
Works for me:

```
CARDANO_NODE_SOCKET_PATH=example/node-bft2/node.sock cardano-cli query tip --testnet-magic 42
{
    "epoch": 0,
    "now": 94,
    "tipTime": 94,
    "hash": "5e2d577395bdb1e996eb3754dced268f9dbe3e50ed50aca131100ddcb2a7ab32",
    "slot": 471,
    "block": 30,
    "era": "Mary",
    "syncProgress": "100.00"
}
```

**Review by newhoggy — APPROVED** [review]

File: scripts/README.md (hunk 1 above)
Comment (Jimbo4350) [reply scripts/README.md:22]: :+1: I'll make the change

**Comment by Jimbo4350** [comment]
bors merge

**Pull request merged**